packages/node-cli/src/validation/InteractiveAnswersValidator.js

```diff
@@ -181,6 +181,14 @@ module.exports = {
 		return !fileSystemService.fileExists(path.join(parentFolderPath, filenameWithExtension))
 			? VALIDATION_RESULT_SUCCESS
 			: VALIDATION_RESULT_FAILURE(NodeTranslationService.getMessage(ANSWERS_VALIDATION_MESSAGES.FILE_ALREADY_EXISTS));
+	},
+
+	validateFolderAlreadyExists(path) {
```

Comment:
I think the name should be, `validateFolderDoesNotExist` .
I have seen also that the previous validate function `validateSuiteScriptFileAlreadyExists`  which I did create is also following the same wrong approach. 
I would say, to at least change the one you have created, and feel free to change the other one if you wish.